### PR TITLE
unbound / overview: migrate to duckdb

### PR DESCRIPTION
**Pull request by swhite2**
Moves the reporting backend from sqlite to duckdb.

Since, unlike sqlite, duckdb does not support concurrency, a wrapper has been written to synchronize access to the database. Refer to the class itself for further documentation.

If you were on the master branch and had unbound statistics enabled prior to this commit, you should `rm -f /var/unbound/data/unbound.sqlite` and restart Unbound.

### CODE REVIEW

**Comment by swhite2** [comment]
> I'm missing package dependencies here...

That's the main reason this PR is still in draft state.

> also what other python do we directly depend on? pandas and numpy or more?

If we're talking direct dependencies, only duckdb, pandas and numpy.

File: src/www/services_unbound.php [hunk 1]
```diff
@@ -40,7 +40,6 @@
     $pconfig = array();
     // boolean values
     $pconfig['enable'] = isset($a_unboundcfg['enable']);
-    $pconfig['stats'] = isset($a_unboundcfg['stats']);
```
Comment (fichtner) [inline src/www/services_unbound.php:43]: me like :)

**Review by fichtner — APPROVED** [review]
looks good on paper, can give it a try tomorrow

**Comment by swhite2** [comment]
> @swhite2 added deps in [afa13df](https://github.com/opnsense/core/commit/afa13df34d30a4fe56a9996a1cfae7f39f14f5d6) 

Thanks!

> -- looking at it "DNS" might be too vague so I'd suggest "Unbound DNS" as reporting entry. Also the Unbound control 
> widget should probably removed if we have this anchored in reporting section. People are probably confused what it will do > (reload stats? reload stats collector? but not unbound itself).

Agreed, within the same context I named it "Unbound DNS" in the reporting settings toggle for the same reason, will change it accordingly for the menu item as well. Thanks for the feedback :)

**Comment by fichtner** [comment]
PS: works fine from here, nice work!

**Pull request merged**